### ISSUE TITLE
hey

### ISSUE THREAD

**Issue by subtoLcLc** [3kh0/discord-proxy#4]
@3kh0 heyyyy

**Comment by 3kh0** [comment]
whats up?

**Comment by subtoLcLc** [comment]
hey can i ask u something

**Comment by 3kh0** [comment]
> hey can i ask u something

Ask away!

**Comment by subtoLcLc** [comment]
can u make all the papas games unblocked and without flash plesssssssssss

**Comment by subtoLcLc** [comment]
u there?

**Comment by subtoLcLc** [comment]
@3kh0

**Comment by 3kh0** [comment]
> can u make all the papas games unblocked and without flash plesssssssssss

yeah, I am working on that. If you want you can join my discord, I post updates there!

**Comment by subtoLcLc** [comment]
i cant :( im on school ipad
but i can prob find unblocked discord

**Comment by subtoLcLc** [comment]
thanks for working on the papas games

**Comment by subtoLcLc** [comment]
can you tell me your email or gmail so i can contact you better?

**Comment by 3kh0** [comment]
> can you tell me your email or gmail so i can contact you better?

sure!

here is my email: echo-the-coder@tuta.io

looking forward to meeting with you!

**Comment by subtoLcLc** [comment]
hello
